frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualizationStepList.scss

```
@@ -20,13 +20,21 @@
 
   &__step {
     display: flex;
-    margin-bottom: 10px;
+    margin-bottom: var(--pf-global--spacer--xs);
+
+    &.--task-run {
```

nit: If following BEM, a block or element modifier should still be qualified with the full name.
```suggestion
    &--task-run {
```

---

Oh yeah... I guess that is true 🤔 I think classnames doesn't roll with that kind of logic, but I should adjust.

---

This is why I also prefer the convention of `is-...`.